src/app/static/src/app/components/genericChart/utils.ts

```diff
@@ -29,8 +29,13 @@ export function genericChartColumnsToFilters(columns: GenericChartColumn[], filt
 
 export function numeralJsToD3format(numeralJsFormat: string) {
     // Convert hintr metadata format (which are numeralJs style) to d3 format to be used by Plotly
-    // We currently support only numeric and percentage formats, and will return empty string for any other
-    // formats received, for default formatting in Plotly.
+    // We currently support only numeric, large number, and percentage formats, and will return 
+    // empty string for any other formats received, for default formatting in Plotly.
+
+    if (numeralJsFormat === "0,0") {
+        return ","
+    }
+
     const regex = /^0(\.0+)?(%)?$/; //This will always return three matches
```

Comment:
```suggestion
    const regex = /^0(\.0+)?(%)?$|^0(,0+)/; //This will always return four matches
```

Reply:
then add below snippet after the nullable check

```
    if (match[3] !== undefined) {
        return ",";
    }
```

Reply:
Could do this - but I think at that point it would be good to comment the regex a little to explain the two parts of it. Regexes are pretty horrible to read..